upup/pkg/fi/task.go

```diff
@@ -57,6 +57,15 @@ func (c *ModelBuilderContext) AddTask(task Task) {
 	c.Tasks[key] = task
 }
 
+func (c *ModelBuilderContext) HasTask(task Task) bool {
```

Comment:
I still want to keep this function in - will be useful when developing / debugging